SECURITY.md

```diff
@@ -0,0 +1,31 @@
+# Critical bugs and security issues 💥
+
+If you're here because you're trying to figure out how to notify us of a security issue, go to [Discord](https://discord.gg/VNyAnw7e), and alert:
```

Comment:
We have an official security.md for wasmd https://github.com/CosmWasm/wasmd/blob/master/SECURITY.md

It would be good to keep this in line with the other one. It currently has one official email, and we are working on securely distributing a GPG key to a few different team members so we can accept encypted messages as well.

I would not recommend reporting bugs on discord.